core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -45,8 +45,8 @@ oppia.directive('questionsList', [
       bindToController: {
         getSkill: '&skill',
         getTopicId: '&topicId',
-        getQuestionSummaries: '=',
-        fetchQuestionSummaries: '=',
+        getTopicSkillIds: '&topicSkillIds',
```

Comment:
Instead of the above three fields why not just pass the list of skill IDs in, so that this directive is agnostic of whether it's in the skill or topic page?

Reply:
Displaying of questions list in topic editor and skill editor is different. For example, in topic editor, select skill modal is shown before creating questions. In skill editor, there is no skill descriptions column because we know it belongs to this skill. That is why that information is still needed.

Reply:
That is different - that is a check to ensure we do not fetch questions for more than 3 skill ids at a time. But a question can be linked to more than 3 skill ids

Reply:
I simplified the skill editor part, but I left getTopicId because topicId is being used for accepting and rejecting questions, which is in the suggestions project I think.